keyformat/key_format.go

```diff
@@ -105,6 +106,13 @@ func (kf *KeyFormat) Key(args ...interface{}) []byte {
 	return kf.KeyBytes(segments...)
 }
 
+func (kf *KeyFormat) NodeKey(nodeKey int64) []byte {
```

Comment:
I can't find usages of this function anywhere, am I missing something?

Reply:
good catch